### ISSUE TITLE
Unable to download packages from NuGet official package source when use protocolVersion 2 and 3 simultaneously in nuget.config

### ISSUE THREAD

**Issue by deepakaravindr** [NuGet/Home#1283]
@StasPerekrestov

Unable to download packages from NuGet official package source

Steps to reproduce:
1. Use protocolVersion="3" for NuGet official package source in nuget.config

``` xml
<add key="NuGet official package source" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```
1. Add own repository 

``` xml
<add key="My nuget repository" value="http://nuget.myserver.com/nuget/" />
```
1. Remove packages folder
2. Clean %AppData%..\Local\NuGet\Cache

Result:
Nuget.exe doesn't download packages from the official source

This was moved from https://github.com/NuGet/NuGet.CommandLine/issues/15

**Comment by yishaigalatzer** [comment]
You need to use a newer version of nuget.exe (2.8.6 or even better 3.2).

A beta release of 3.2 is available here -  http://dist.nuget.org/win-x86-commandline/v3.1.0-beta/nuget.exe

We plan an update later this week.

**Comment by StasPerekrestov** [comment]
Thank you!

**Comment by yishaigalatzer** [comment]
Update - We released 3.2-RTM

You can find all current downloads at https://dist.nuget.org/index.html
